subsys/bluetooth/audio/audio_iso.h

```diff
@@ -0,0 +1,62 @@
+/* @file
+ * @brief Internal APIs for Audio ISO
+ *
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/iso.h>
+#include <zephyr/bluetooth/audio/audio.h>
+
+/** Audio ISO Type */
+enum bt_audio_iso_type {
+	BT_AUDIO_ISO_TYPE_UNICAST = BIT(0),
+	BT_AUDIO_ISO_TYPE_BROADCAST = BIT(1),
+	BT_AUDIO_ISO_TYPE = BT_AUDIO_ISO_TYPE_UNICAST |
+			    BT_AUDIO_ISO_TYPE_BROADCAST,
+} __packed;
```

Comment:
I pack the enum to decrease it's size from 4 to 1 byte. I'll move sink_path_cc and source_path_cc to be placed after that, so this might save few bytes :)

Reply:
Generally (at least in the BT subsystem) we only pack enums if they represent a value that we want to send over air (e.g. if we want an enum for opcodes that we need to ensure is 8-bit). 

In all other cases, we avoid packing it so that we can leave that up to the compiler